fuzz/fuzz_targets/table_ops.rs

```diff
@@ -0,0 +1,55 @@
+#![no_main]
+
+use libfuzzer_sys::arbitrary::{Arbitrary, Unstructured};
+use libfuzzer_sys::{fuzz_mutator, fuzz_target, fuzzer_mutate};
+use mutatis::Session;
+use postcard::{from_bytes, to_slice};
+use rand::{Rng, SeedableRng};
+use wasmtime_fuzzing::generators::table_ops::TableOps;
+use wasmtime_fuzzing::oracles::table_ops;
+
+fuzz_target!(|input: (u64, TableOps)| {
```

Comment:
We don't want the fuzz target to take structured types directly, as those get generated via `Arbitrary`, and we want to do our serialization- and mutation-based thing here.

Instead we should do something like this:

```rust
fuzz_target!(|data: &[u8]| {
    let Ok((seed, ops)) = postcard::from_bytes::<(u64, TableOps)>(data) else {
        return;
    }

    // As before: Make an RNG from the seed, fill the buf with random
    // data, make an `Unstructured`, call `Config::arbitrary_take_rest`,
    // and finally invoke `oracles::table_ops(config, ops)`.
});
```

Aside: once this PR lands, we should probably remove the `Arbitrary` implementation for `TableOps` as we won't be using it anymore and removing it will make usage footguns like this one disappear.